### DISCUSSION TITLE
Is there a built-in way to make a window follow you?

### DISCUSSION THREAD

**Discussion by bootkernel** [StereoKit/StereoKit#857] [General]
## Describe the feature
I'm searching for a function that replicates the behavior seen in this Hololens demo: [https://youtu.be/uIHPPtPBgHk?t=115](url) in a 6DoF environment.

I was only able to find the pose LookAt function.

## An example use-case
I'd like to see how to create a window that smoothly (using lerp) looks at and follows the user.

**Comment by maluoi** [comment]
Not built in, but pretty easy! Basically something along these lines:

```csharp
Pose windowPose = Pose.Identity;
Pose destPose   = Pose.Identity;

void Step() {
    destPose   = new Pose(Input.Head.Ray.At(1), Quat.FromAngles(0,180,0) * Input.Head.orientation);
    windowPose = Pose.Lerp(windowPose, destPose, Time.Frame * 8);
    
    UI.WindowBegin("Window", ref windowPose);
    UI.WindowEnd();
}
```

**Reply by bootkernel** [reply]
Thank you!
